lib/api_callable.js

```diff
@@ -190,19 +190,32 @@ function retryable(aFunc, retry, otherArgs) {
   return function retryingFunc(argument, callback) {
     var canceller;
     var timeoutId;
-    var error = new Error('Retry total timeout exceeded before any' +
-        'response was received');
+    var error = new Error();
```

Comment:
Let's remove this line.  an `Error` actually stores the callstack of methods, and creating an Error here would look a bit confusing to the users.